api/src/pcapi/connectors/api_entreprises.py

```diff
@@ -16,9 +16,11 @@ class ApiEntrepriseException(Exception):
 
 
 def get_by_offerer(offerer: "Offerer") -> dict:
-    response = requests.get(
-        f"https://entreprise.data.gouv.fr/api/sirene/v3/unites_legales/{offerer.siren}", verify=False
-    )
+    try:
+        response = requests.get(f"https://entreprise.data.gouv.fr/api/sirene/v3/unites_legales/{offerer.siren}")
+    except (requests.ConnectionError, requests.HTTPError, requests.Timeout) as exc:
```

Comment:
Ne devrait-on pas être plus large dans les exceptions ?

```suggestion
    except requests.RequestException as exc:
```

Ça inclut entre autres les 3 exceptions que tu listais, mais aussi `SSLError` et d'autres trucs qui sont sûrement plus improbables, mais qui ne coûte pas cher d'attraper (à mon avis). Qu'en penses-tu ?

Reply:
Je me suis posé la question et je pense que je ferai ça dans un second temps avec un warning séparé pour les alertes OPS.